contrib/Shopify/sarama/sarama.go

```diff
@@ -176,7 +177,8 @@ func (p *asyncProducer) Errors() <-chan *sarama.ProducerError {
 
 // WrapAsyncProducer wraps a sarama.AsyncProducer so that all produced messages
 // are traced. It requires the underlying sarama Config so we can know whether
-// or not sucesses will be returned.
+// or not successes will be returned. Tracing requires at least sarama.V0_11_0_0
```

Comment:
Can we improve this documentation to note that successes must be returned in order to know the offset/partition?